src/plugins/editor/components/editor.jsx

```diff
@@ -240,7 +240,7 @@ export default function makeEditor({ editorPluginsToRun }) {
       editor.setReadOnly(readOnly)
     }
 
-    componentWillMount() {
+    UNSAFE_componentWillMount() {
```

Comment:
should not use React 16 method here.

Reply:
I supposed we could disable eslint for these two lines that are using the deprecated lifecycle methods or I could find a way to use a different lifecycle method that will work without any eslint errors. What do you think @tim-lai?

Reply:
Ideally, we shouldn't disable if we don't need to so I'll start off with refactoring these lifecycle methods.

Reply:
for `componentWillMount`, it is safe to move to `componentDidMount`.

Reply:
Fixed